frontend/ui/widget/filechooser.lua

```diff
@@ -495,6 +495,14 @@ function FileChooser:getNextFile(curr_file)
     return next_file
 end
 
+function FileChooser:selectAllFilesInFolder()
+    for _, item in pairs(self.item_table) do
+        if item.bidi_wrap_func == BD.filename then
```

Comment:
I guess this is to detect if item is a file - and not a directory ?
But it's a bit ugly (and would really need a comment). Can't you just add a property where item is made, so you can use item.is_file or item.is_filemanager_file ?

Reply:
Or even just the regular `lfs.attributes(item.path, "mode") == "file" then` - this is so rarely (and willingly) used that we don't care about optimizing it.

Reply:
Thanks, fixed with `is_file`, short, quick and beautiful.